Automattic/wp-calypso
# Domain management i1: rework sorting logic

## Proposed Changes

* Review and update sorting logic
  - Remove secondary sort functions which won't make sense to the user
  - Fix `owner` and `site` sorting logic
  - Add some tests 

## Testing Instructions

- Try it out at `/domains/manage` and `/domains/manage/%s`
- Do a code review and see if you agree that most columns now only use `getSimpleSortFunctionBy`



## Pre-merge Checklist
